cmd/minishift/cmd/update.go

```diff
@@ -73,30 +75,60 @@ func runUpdate(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
 	}
 
-	latestVersion, err := update.LatestVersion()
+	if versionFlag == "" {
+		updateToLatest(localVersion)
+	} else {
+		version, _ := semver.Make(versionFlag)
+		if err := update.Update(version); err != nil {
+			atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
+		}
+		fmt.Printf("\nUpdated successfully to Minishift version %s.\n", version)
+	}
+}
```

Comment:
@budhrg this is something not look clean, till now what we were doing is updating directly to latest update version but now we just add a verison parameter so for that our base logic doesn't change. And your `updateToLatest` function change to `updateToVersion` which can be latest.

Something like below.

```golang
	currentVersion, err := update.CurrentVersion()
	if err != nil {
		atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
	}
	versionToUpdate, err := update.LatestVersion()
	if versionFlag != "" {
		versionToUpdate, err = semver.Make(versionFlag)
	}
	updateToVersion(currentVersion, versionToUpdate)
```